apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/ValueConversion.cs

```diff
@@ -76,21 +76,16 @@ public static object ConvertToClrType(this Value wireValue, V1.Type spannerType,
                 return Convert.ToUInt16((long) ConvertToClrTypeImpl(wireValue, spannerType, typeof(long)));
             }
 
-            if (targetClrType == typeof(char))
+            if (targetClrType == typeof(sbyte))
```

Comment:
no char needs special handling and cannot leverage the long codepath.  it was removed on purpose.